demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -203,10 +204,45 @@ def _search_for_pack_items(pack_id: str, items_list: list) -> list:
         """
         return list(filter(lambda s: next(iter(s.values())).get('pack') == pack_id, items_list))
 
+    @staticmethod
+    def _is_item_match(item_details: dict, base_condition: bool, exclude_ignored_dependencies: bool, marketplace: str) -> bool:
+        """
+        Whether the item matches the criterias:
+            * matches the base condition.
+            * relevant to the marketplaces server versions.
+            * not in an excluded pack.
+            * part of the desired marketplce.
+
+        Args:
+            item_details: item info from the ID set.
+            base_condition: basic check of relevance, for example item ID is matched.
+            exclude_ignored_dependencies: Determines whether to include unsupported dependencies or not.
+            marketplace: The dependency calculation desired marketplace.
+
+        Returns:
+            bool
+        """
+        id_set_item_version = Version(item_details.get('toversion', DEFAULT_CONTENT_ITEM_TO_VERSION))
```

Comment:
Shouldn't it be "to_version" instead of "toversion"?

Reply:
No, This is how we save it in the ID set.